.github/workflows/security.yml

```diff
@@ -0,0 +1,69 @@
+name: Security-check
+
+on:
+  pull_request: # change to push before merging code
+    branches:
+      - dev
+    paths:
+      - "**.py"
+      - ".github/workflows/*.yml"
+
+jobs:
+  python-tests:
```

Comment:
Hmm..I think  this is the security one

Reply:
Yeah, I guess I interchanged the names, changing that